README.rst

```diff
@@ -20,7 +20,7 @@ Intro
 Django-Estimators helps persist and track machine learning models (aka estimators) and datasets.
 
 
-This library provides a series of proxy objects that wrap common python machine learning objects and dataset objects.  As a result, this library can be used to version, track progress and deploy models.  It's highly extensible and can be used with almost any python object (scikit-learn, numpy arrays, modules, methods).
+This library provides a series of proxy objects that wrap common python machine learning objects and dataset objects.  This library versions, track progress and deploy models.  It's highly extensible and can be used with most any python object (scikit-learn, numpy arrays, modules, methods).
```

Comment:
Couple spelling edits, but for the first, maybe there's a better suggestion.

Instead of `This library versions, track progress and deploy models.` do you think we can do `These objects help version and track models, for the purposes of productionizing, deploying, and benchmarking models`

`most any python` and `with most python`